modules/wri_subpage/templates/wri-subpage-columns.html.twig

```diff
@@ -45,8 +45,8 @@
     {% endif %}
     {# end if content.hero #}
 
-    {% if content.category or content.share %}
-      {% if (content.hero.field_toc is not empty) and (content.hero.field_toc['#value'] != 'microsite') %}
+    {% if content.field_toc['#value'] %}
+      {% if (content.hero.field_toc is not empty) %}
         <div class="mobile__toc grid">
```

Comment:
Somehow something in here (I'm guessing) is making us lose our underline and "Topic" section on pages like this one on desktop:
https://pr-1223-wriflagship.pantheonsite.io/research/achieving-replacement-level-fertility

compare: https://www.wri.org/research/achieving-replacement-level-fertility

Same kind of thing here:
https://pr-1223-wriflagship.pantheonsite.io/initiatives/african-forest-landscape-restoration-initiative-afr100

I think you've got two new opening if statements that need to be closed before `{% if content.category or content.share %}` below.

Reply:
@mariacha Ah, yes - was a little to heavy-handed in trying to get the subpages working on microsite subpages. Fixed!